helm/templates/client/clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: ols-proxy-backend-register
```

Comment:
i think we should drop this binding or move it to some sort of examples folder.  it's not part of the core things that are actually needed when installing OLS